llvm/docs/GitHub.rst

```diff
@@ -274,8 +264,24 @@ Your options are as follows:
 
    This approach allows GitHub to display clean, incremental diffs for each PR
    in the stack, making it much easier for reviewers to see what has changed at
-   each step. Once `feature_1` is merged, you can rebase and re-target
-   `feature_2` to `main`.
+   each step. Once `feature_1` is merged, GitHub will automatically rebase and
+   re-target your branch `feature_2` to `main`. For more complex stacks, you can
+   perform this step using the web interface.
+
+   This approach requires
+   `Commit Access <https://llvm.org/docs/DeveloperPolicy.html#obtaining-commit-access>`_.
+   However, if you are involved with a Stacked PR, there's a good chance you'll
+   be granted access.
```

Comment:
[nit] The paragraph you linked discusses the process of obtaining commit access, rather than providing a general definition of what "commit access" means. I suggest updating the link title to reflect that more accurately.

Also, regarding the sentence:

> However, if you are involved with a Stacked PR, there's a good chance you'll be granted access.

This phrasing could be misleading. In the LLVM project, needing stacked PRs is not itself a sufficient criterion for being granted commit access. We should avoid suggesting a causal link where there isn’t one.

I suggest keeping this simple and factual: "This approach requires commit access: <link to how obtain the LLVM commit access>".

I hope that this makes sense.

Reply:
Hey Andrzej,

Good points, and thanks for your review!

I've kept the link and now it refers on how to obtain commit access. – or do you think it's better to remove it?